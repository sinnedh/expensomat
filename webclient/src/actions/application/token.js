export const setToken = (token) => {
  return {
    type: 'APPLICATION:SET_TOKEN',
    token,
  }
}
