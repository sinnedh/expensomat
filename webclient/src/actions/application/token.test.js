import * as actions from './token'

describe('APPLICATION', () => {
  const token = 'ABCD123'

  it('handles setToken', () => {
    const action = actions.setToken(token)
    expect(action).toMatchSnapshot()
  })
})
