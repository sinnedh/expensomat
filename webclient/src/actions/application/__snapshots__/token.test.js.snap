// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APPLICATION handles setToken 1`] = `
Object {
  "token": "ABCD123",
  "type": "APPLICATION:SET_TOKEN",
}
`;
